INSTALL.rst

```diff
@@ -44,6 +44,18 @@ From source
   git clone https://github.com/k2-fsa/k2.git
   cd k2
   python3 setup.py install
+  
+From source (with CUDA support)
+===========
+
+.. code-block:: bash
+
+  git clone https://github.com/k2-fsa/k2.git
+  cd k2
+  export K2_CMAKE_ARGS="-DK2_WITH_CUDA=ON"
```

Comment:
```suggestion
  export K2_CMAKE_ARGS="-DK2_WITH_CUDA=ON -DCMAKE_BUILD_TYPE=Release"
```
It defaults to `Debug` build.

Reply:
Agreed, making explicitly "Release" is unambiguous.